ultravox/tools/infer_api.py

```diff
@@ -23,6 +24,7 @@ def infer(
         sample: datasets.VoiceSample,
         max_tokens: Optional[int] = None,
         temperature: Optional[float] = None,
+        past_key_values: Optional[Union[Tuple, transformers.cache_utils.Cache]] = None,
```

Comment:
I'm confused why you need this. Even `LocalInference.infer` doesn't take `past_key_values`.

Reply:
Good catch. `past_key_values` was removed from `LocalInference.infer` as we put `history` implementation inside `LocalInference`, so `past_key_values` is no longer needed.